docs/general/installation/truenas.md

```diff
@@ -0,0 +1,534 @@
+The SMB share mount does not include ACL protections at this time. Permissions are currently limited to the permissions of the user that mounted the share.
+Alternate data streams (metadata), finder colors tags, previews, resource forks, and MacOS metadata are stripped from the share along with filesystem permissions, but this functionality is undergoing active development and implementation planned for a future TrueNAS SCALE release.
+
+- Note that if you want to take advantage of Jellyfin's built-in feature of **real-time media scanning**, you need to mount your media directly with a **Host Path** as SMB connections do not support this feature.
+
+For all types, enter a **Mount Path** to be used within the Jellyfin container.
+
+- For example, the local **Host Path** /mnt/tank/video/movies could be assigned the **Mount Path** /media/movies.
+  - With this example, you would browse to `/media/movies` in Jellyfin to see the contents of `/mnt/tank/video/movies` on your SCALE server.
+
+<details>
+  <summary>Additional Storage Fields</summary>
+
+  <table>
+    <thead>
+      <tr>
+        <th>Type</th>
+        <th>Field</th>
+        <th>Description</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>All</td>
+        <td>Mount Path</td>
+        <td>The virtual path to mount the storage within the container.</td>
+      </tr>
+      <tr>
+        <td>Host Path</td>
+        <td>Host Path</td>
+        <td>The local path to an existing dataset on the System.</td>
+      </tr>
+      <tr>
+        <td>ixVolume</td>
+        <td>Dataset Name</td>
+        <td>The name for the dataset the system creates.</td>
+      </tr>
+      <tr>
+        <td>SMB Share</td>
+        <td>Server</td>
+        <td>The server for the SMB share.</td>
+      </tr>
+      <tr>
+        <td>SMB Share</td>
+        <td>Share</td>
+        <td>The name of the share.</td>
+      </tr>
+      <tr>
+        <td>SMB Share</td>
+        <td>Domain (Optional)</td>
+        <td>The domain for the SMB share.</td>
+      </tr>
+      <tr>
+        <td>SMB Share</td>
+        <td>Username</td>
+        <td>The user name used to access the SMB share.</td>
+      </tr>
+      <tr>
+        <td>SMB Share</td>
+        <td>Password</td>
+        <td>The password for the SMB share user.</td>
+      </tr>
+      <tr>
+        <td>SMB Share</td>
+        <td>Size (in Gi)</td>
+        <td>The quota size for the share volume. You can edit the size after deploying the application if you need to increase the storage volume capacity for the share.</td>
+      </tr>
+    </tbody>
+  </table>
+</details>
+
+### Resource Configuration Settings
+
+![Resource Limits](/images/docs/install-truenas-16.png)
+
+You can customize limits on the CPU and memory allocated to the container Jellyfin will reside in.
+
+- **CPUs** expects a value in **number of threads** to assign as a max CPU thread limit.
+  - The default is **2** which means the container will use 2 CPU threads at most.
```

Comment:
2 is asking for trouble. Should just tell users to set it to number of threads their CPU has